index.md

```diff
@@ -2,131 +2,201 @@
 layout: spec
```

Comment:
Maybe I'm missing something, but I don't think this is linked from the main page AKA `README.md`.  Perhaps a good way to structure the linking would be to structure the README like this:

- Using
    - HTML/JavaScript
    - Markdown/Jekyll
    - Walk through example

The walk through example could link to the meta project that you added.

Reply:
Makes sense! Yes, I missed to include this demo in the README.

Reply:
Actually I had included it at the top of the README. I've also referenced it at the end of the "Jekyll + MarkDown" section. (I wasn't sure if I had enough content to create a "walk through example" section.